app/win/BUILD.gn

```diff
@@ -160,3 +160,10 @@ copy("dist_resources") {
   ]
 }
 
+windows_manifest("electron_manifest") {
```

Comment:
I'm a little confused because this isn't declared as a dep anywhere. I think it should replace `"//build/win:default_exe_manifest"` in `:electron_app`. I think it also needs `default_compatibility_manifest`, `common_controls_manifest` and `as_invoker_manifest` to match the current atom.manifest file, but